Tools/cases_generator/generators_common.py

```diff
@@ -165,16 +165,24 @@ def error_if(
         storage: Storage,
         inst: Instruction | None,
     ) -> bool:
-        self.out.emit_at("if ", tkn)
         lparen = next(tkn_iter)
-        self.emit(lparen)
         assert lparen.kind == "LPAREN"
         first_tkn = tkn_iter.peek()
-        emit_to(self.out, tkn_iter, "COMMA")
```

Comment:
Wouldn't it be simpler to just add an unconditional `ERROR()` macro to the DSL?

Reply:
That wouldn't prevent anyone writing `ERROR_IF(true, ...)` so we'd still need to handle it.